fastlane/lib/fastlane/actions/docs/frame_screenshots.md

```diff
@@ -215,6 +215,46 @@ The `keyword.strings` and `title.strings` are standard `.strings` file you alrea
 - These `.strings` files **MUST** be utf-8 (UTF-8) or utf-16 encoded (UTF-16 BE with BOM). They also must begin with an empty line. If you are having trouble see [issue #1740](https://github.com/fastlane/fastlane/issues/1740)
 - You **MUST** provide a background if you want titles. _frameit_ will not add the tiles if a background is not specified.
 
+### Screenshot orientation
+
+By default _frameit_ adds a frame to your screenshot based on an orientation you took it. For a portrait (vertical orientation) it is going to add portrait frame and for a landscape (horizontal orientation) - landscape left.
+
+If you'd like to get a frame in a landscape right for your horizontal screenshots, you need to add `force_landscaperight` at the end of your screenshot's filename. 
+
+### `force_orientation_block`
+
+In a case you don't want to rename your screenshots or default behaviour does not meet your requirements, there's a way to customise it. In order to do so, you need to set `force_orientation_block` parameter.
```

Comment:
```suggestion
If the default behavior doesn't fit your needs and you don't want or can't rename your screenshots, you can customize _frameit_'s orientation behavior by setting a `force_orientation_block` parameter.
```

Reply:
You can accept this change here in the GitHub UI and it will automatically be added. (Caution: You have to fetch if you have a local checkout before committing again, otherwise you might get a merge conflict)